website/views.py

```diff
@@ -292,126 +373,189 @@ def answer_comment(request):
+                    subject, '',
+                    sender_email, to,
+                    bcc=[bcc_email],
+                    headers={"Content-type": "text/html;charset=iso-8859-1"}
+                )
+                email.attach_alternative(message, "text/html")
+                email.content_subtype = 'html'  # Main content is text/html
+                email.mixed_subtype = 'related'
+                email.send(fail_silently=True)
+
+            comment_creator = AnswerComment.objects.order_by('-date_created').values(
+                'uid').filter(answer=answer, is_active=True).exclude(uid=request.user.id).distinct()
+            if comment_creator:
+
+                notification = Notification()
+                notification.uid = comment_creator[0]['uid']
+                notification.qid = answer.question.id
+                notification.aid = answer.id
+                notification.cid = comment.id
+                notification.save()
 
-            # notifying other users in the comment thread
-            uids = answer.answercomment_set.filter(answer = answer).values_list('uid', flat = True)
-            answer_comments = answer.answercomment_set.filter(answer = answer)
+                sender_name = "FOSSEE Forums"
+                sender_email = settings.SENDER_EMAIL
+                bcc_email = settings.BCC_EMAIL_ID
+                subject = "FOSSEE Forums - {0} - Comment has a reply".format(
+                    answer.question.category)
+                to = [get_user_email(comment_creator[0]['uid'])]
+                message = """
+                    A reply has been posted on your comment.<br><br>
+                    <b>Title:</b> {0}<br>
+                    <b>Category:</b> {1}<br>
+                    <b>Link:</b> {2}<br><br>
+                    Regards,<br>
+                    FOSSEE Team,<br>
+                    FOSSEE, IIT Bombay
+                    <br><br><br>
+                    <center><h6>*** This is an automatically generated email, please do not reply***</h6></center>
+                    """.format(
+                    answer.question.title,
+                    answer.question.category,
+                    settings.DOMAIN_NAME + '/question/' + str(answer.question.id) + "#answer" + str(answer.id)
+                )
 
-            comment_creator_emails = []
-            for c in answer_comments:
-                comment_creator = c.user()
+                email = EmailMultiAlternatives(
+                    subject, '',
+                    sender_email, to,
+                    bcc=[bcc_email],
+                    headers={"Content-type": "text/html;charset=iso-8859-1"}
+                )
+                email.attach_alternative(message, "text/html")
+                email.content_subtype = 'html'  # Main content is text/html
+                email.mixed_subtype = 'related'
+                email.send(fail_silently=True)
+
+            comment_set = []
```

Comment:
The below code looks similar to the code on the top.
So can we have a function for this so that the redundancy is reduced.

Reply:
functions has been declared
issue solved